nvidia-temp@sophie-la-li/files/nvidia-temp@sophie-la-li/applet.js

```diff
@@ -23,6 +25,16 @@ NvGpuTempApplet.prototype = {
         this.settings.bindProperty(Settings.BindingDirection.IN, 'show-unit-letter', 'showUnitLetter', () => this.refresh(), null);
         this.settings.bindProperty(Settings.BindingDirection.IN, 'use-fahrenheit', 'useFahrenheit', () => this.refresh(), null);
 
+        if (!GLib.find_program_in_path("nvidia-smi")) {
+            let icon = new St.Icon({ 
+                icon_name: 'error',
+                icon_type: St.IconType.FULLCOLOR,
+                icon_size: 36 
+            });
+            Main.criticalNotify(_('NVidia GPU Temperature Indicator: Error'), _('It seems that nvidia-smi is not installed.'), icon);
```

Comment:
I wasn't sure if it was intentional or not but the proper spelling of the proper noun is Nvidia.

Reply:
done